### ISSUE TITLE
20$: Warn on [] or () following newline without semicolon

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#693]
The following code is misleading:
```js
client.commands = new Collections()
["command", "event"].forEach(addCommand)
```
quick-lint-js should report a warning that a semicolon is required if the second line is meant to be its own statement.

**Comment by strager** [comment]
This issue was already filed: #431
